yellowbrick/cluster/elbow.py

```diff
@@ -496,7 +505,7 @@ def kelbow_visualizer(
         locate_elbow=locate_elbow,
         **kwargs
     )
-    oz.fit(X, y)
+    oz.fit(X, y, **fit_kwargs)
```

Comment:
I think the best way to test this would be to add two tests to `tests/test_cluster/test_elbow.py`, one to make sure we're using the kwargs properly now, and one to make sure that the quick method correctly sorts the kwargs, e.g.:

```python

def test_sample_weights(self):
    """
    Test that passing in sample weights correctly influences the clusterer's fit
    """
    X, y = load_nfl()

    visualizer = KElbowVisualizer(
        MiniBatchKMeans(random_state=37), k=5, timings=False, locate_elbow=False
    )
    visualizer.fit(X, sample_weight=y)  # is this right?
    visualizer.finalize()
    expected = np.array(
        [
            # fill in ...
        ]
    )
    assert_array_almost_equal(visualizer.k_scores_, expected)


def test_quick_method_params(self):
    """
    Test the quick method correctly consumes the user-provided parameters
    """
    X, y = load_nfl()
    custom_title = "My custom title"
    model = KMeans(3, random_state=13)
    oz = kelbow_visualizer(model, X, sample_weight=y, title=custom_title)
    assert oz.title == custom_title
```

Reply:
Thanks for this! The quick method and helper tests look pretty straightforward, and I can work on them.

Regarding sample weights test: I was thinking about something like this, but I'm not sure how to craft the data in a way to make the test pass. In this case the sample_weight should be a vector describing the importance of each observation when computing centroids. My original idea was to craft a dataset with, say K=5 clusters, but tweak the sample_weights in a way that we get a different number of clusters according to the elbow. Not sure what is the best approach. Let me know what you think!

Reply:
That sounds great!